### ISSUE TITLE
Bunch fails to launch Hazeover

### ISSUE THREAD

**Issue by Stever888** [ttscoff/bunch#139]
Every morning, I have a Bunch that launches and opens the book I'm working on. The Bunch launches Hazeover to help me focus. The launching of Hazeover has been failing however. Here's my Bunch, the Log, and my System Preferences "Privacy \& Security" screenshot.

# My Bunch

\---
open on: MTWThF 06:25 am
\# only auto-open on my work machine
schedule if: _my-real-machine-UUID_
menu order: 5
\---

Scrivener
\- ~/Dropbox/Apps/Scrivener/manuscript.scriv

firefox:_writing group URL_

Firefox\_ ~3
HazeOver
\* tell app "HazeOver" to set enabled to true
\* tell app "HazeOver" to set intensity to 90

\@Scrivener ~5

# The Log 

```
2021-08-24 06:25:03: 🟡[GBK] Scheduled to open weekly starting 8/25/21, 06:25
2021-08-24 06:57:56: 🔴{
    NSAppleScriptErrorAppName = HazeOver;
    NSAppleScriptErrorBriefMessage = "Not authorized to send Apple events to HazeOver.";
    NSAppleScriptErrorMessage = "Not authorized to send Apple events to HazeOver.";
    NSAppleScriptErrorNumber = "-1743";
    NSAppleScriptErrorRange = "NSRange: {27, 7}";
}
2021-08-24 06:57:57: 🔴{
    NSAppleScriptErrorAppName = HazeOver;
    NSAppleScriptErrorBriefMessage = "Not authorized to send Apple events to HazeOver.";
    NSAppleScriptErrorMessage = "Not authorized to send Apple events to HazeOver.";
    NSAppleScriptErrorNumber = "-1743";
    NSAppleScriptErrorRange = "NSRange: {27, 9}";
}
```

I have my security preferences set correctly in `System Preferences`, however.

<img width="780" alt="Screen Shot 2021-08-24 at 07 00 24" src="https://user-images.githubusercontent.com/10833275/130611440-86f61c50-be3e-4760-95ff-4a8887e0c2af.png">

**Comment by ttscoff** [comment]
Hmmm. Same thing works for me, with the same settings. I assume Bunch has permissions under Accessibility as well?

The only thing I can think of is to nuke your AppleEvents permissions, which would cause any app needing them to present the permission dialog again (which might be a lot, depending on your setup). If you want to try it, though, open Terminal and run `tccutil reset AppleEvents`.

**Comment by Stever888** [comment]
Hi, Brett,

I've reset the permissions. I'll let you know what happens :)

Stever
(he/him)

-- 
iTunes top-10 business podcast: https://itunes.com/getitdoneguy
Start finishing with Get-it-Done Groups: https://www.getitdonegroups.com
Executive coaching: https://SteverRobbins.com
+1 (617) 354-1446

NEED TO REACH ME QUICKLY? Text 617-354-1446 xxx Email won’t do it. I only check once a day.

**Comment by Stever888** [comment]
Hi, Brett,

I nuked the AppleEvents permissions and now Bunch is successfully controlling Hazeover again. Shall I let you know if things change?

Great work with Bunch. It's become one of my most-used utilities (Bunch, Yoink, Better TouchTool, and Default Folder X have largely been game changers. I don't know how I survived without them!)

Stever
(he/him)

-- 
iTunes top-10 business podcast: https://itunes.com/getitdoneguy
Start finishing with Get-it-Done Groups: https://www.getitdonegroups.com
Executive coaching: https://SteverRobbins.com
+1 (617) 354-1446

NEED TO REACH ME QUICKLY? Text 617-354-1446 xxx Email won’t do it. I only check once a day.
